### ISSUE TITLE
Material Tree checklist example - Mistake in node property

### ISSUE THREAD

**Issue by Vevl** [angular/components#20216]
https://github.com/angular/components/blob/8b680837931a4c22734bf1cf4cc076aa200e07aa/src/components-examples/material/tree/tree-checklist/tree-checklist-example.html#L20

Hello, just quick info about this typo.

TodoItemFlatNode doesn't have property "filename". And aria-label property in dom then looks like --> "toggle undefined".

Thank you for awesome job guys.

Best Regards.
